hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/snapshot/SnapshotDiffResponse.java

```diff
@@ -41,6 +41,15 @@ public JobStatusProto toProtobuf() {
     public static JobStatus fromProtobuf(JobStatusProto jobStatusProto) {
       return JobStatus.valueOf(jobStatusProto.name());
     }
+
+    public static JobStatus getJobStatusFromString(String jobStatus) {
```

Comment:
This helper function is not needed. Enum has [valueOf()](https://docs.oracle.com/javase/8/docs/api/java/lang/Enum.html#valueOf-java.lang.Class-java.lang.String-) which can be used directly. 

```
JobStatus.valueOf(jobStatus);
```

You just have to capitalized String value of [jobStatus](https://github.com/apache/ozone/pull/4824/files#diff-46f82d5fbac5dc7aae1fc363811fb88e6bc6c89a2aefafddad90de2f1ff3cfbdR401) here.